### DISCUSSION TITLE
地暖控制器 cubee.airrtc.th123e 未能被识别为温控器实体

### DISCUSSION THREAD

**Discussion by keniji** [XiaoMi/ha_xiaomi_home#209] [Ideas]
想了下这个情况在集成里应该是设计使然，但是在`0.1.2`版本里有了相关的修订，所以还是提个feature request吧…
目前控制器在本集成里是拆分成了数个独立的实体：
![image](https://github.com/user-attachments/assets/5ffffb53-347a-4697-a356-939c7854e929)
贴上隔壁集成提供的温控器截图：
![image](https://github.com/user-attachments/assets/504a1f33-7c87-48be-8c3d-d28e1b86c196)
查了下米物，这个牌子一共有四个类似的型号：
[https://mijia.wiki/products/cubee.airrtc.th123e](url)
[https://mijia.wiki/products/cubee.airrtc.th123w](url)
[https://mijia.wiki/products/cubee.airrtc.th125a](url)
[https://mijia.wiki/products/cubee.airrtc.th125t](url)

另外我比较习惯在一个实体里直接调用属性，而不是使用拆分的实体，所以想问一下有手动写配置的方法（或其他方法，除了在HA里配置template）可以把拆分的实体合并成一个的吗？看了下[README](https://github.com/XiaoMi/ha_xiaomi_home/blob/main/README.md)是没太看懂…

**Comment by SusanPhevos** [comment]
xiaomi_home v0.1.2还不支持温控器显示为Climate Entity，我们今后会支持。感谢提议！

**Reply by keniji** [reply]
好的，那先搞个template climate用用…感谢！

**Comment by earlzz** [comment]
suittc.airrtc.wk168  这个也用不了

**Comment by SusanPhevos** [comment]
请帮忙验证一下 #470 功能是否正常
